polyply/tests/test_gen_params.py

```diff
@@ -112,7 +113,19 @@ def test_gen_params(args_in, ref_file):
         assert int_types_ref == int_types_new
 
         for key in force_field.blocks[ref_name].interactions:
-            print(key)
             for term in force_field.blocks[ref_name].interactions[key]:
-                print(term)
                 assert term in force_field.blocks[args.name].interactions[key]
+
+def test_find_missing_links():
+    fname = TEST_DATA + "/gen_params/ref/P3HT_10.itp"
+    ff = vermouth.forcefield.ForceField("test")
+    meta_mol = MetaMolecule.from_itp(ff, fname, "P3HTref")
+    meta_mol.molecule.remove_edge(39, 45) # resid 7,8
+    meta_mol.molecule.remove_edge(15, 21) # resid 3,4
+    missing = list(find_missing_edges(meta_mol, meta_mol.molecule))
+    assert len(missing) == 2
+    for edge in missing:
+        assert edge["resA"] == "P3HTref"
+        assert edge["resB"] == "P3HTref"
+        assert edge["idxA"] in [3, 4, 7, 8]
+        assert edge["idxB"] in [3, 4, 7, 8]
```

Comment:
`edge["idxA"]` could be always 3 this way. Better to collect them and comparing e.g. sets.